js/ui/data_grid/ui.data_grid.export.js

```diff
@@ -374,6 +374,10 @@ exports.ExportController = dataGridCore.ViewController.inherit({}).include(expor
         return result;
     },
 
+    _needColumnExporting: function(column) {
+        return !column.command && (column.allowExporting || column.allowExporting === undefined);
```

Comment:
What if the allowExporting equal the null value? Maybe to use the `typeUtilis.isDefined()`?